src/v3.rs

```diff
@@ -20,15 +20,14 @@ impl Uuid {
     /// [`NAMESPACE_URL`]: #associatedconstant.NAMESPACE_URL
     /// [`NAMESPACE_X500`]: #associatedconstant.NAMESPACE_X500
     pub fn new_v3(namespace: &Uuid, name: &[u8]) -> Uuid {
-        let mut context = md5::Context::new();
+        let mut hasher = md5::Md5::new();
```

Comment:
Use `use md5::{Md5, Digest};` impoer to avoid duplication of `md5` here? Same goes for SHA-1.